src/login-manager.android.ts

```diff
@@ -2,7 +2,7 @@ import * as application from "tns-core-modules/application";
 import { LoginResponse } from './login-response';
 declare let com: any;
 
-const LOGIN_PERMISSIONS = ["public_profile", "email"];
+const LOGIN_PERMISSIONS = ["public_profile", "email", "user_friends"];
 
```

Comment:
"user_friends" is not a permission needed to do the login. That's why we cannot accept this change. In case you need this permission anyway, you can request it explicitly by using `requestPublishPermissions`. 

Let us know if this works for you.